### PR TITLE
chore(linux): copy Keyman for Linux 15 reference to 14

### PR DESCRIPTION
**Pull request by mcdurdin**
As we auto-generate the reference docs for 15.0, but not 14.0, let's just copy the version 15 docs into 14 so we have them available online.

Query (@ermshiperete): are the docs accurate for 14.0 or have options changed?

@keymanapp-test-bot skip

### CODE REVIEW

File: linux/help/index.md [hunk 1]
```diff
@@ -12,6 +12,14 @@ Need help using Keyman for Linux? In time, this product documentation will grow
...
 * [Installing a keyboard](start/installing-keyboard)
 
+## [Reference](reference/)
+* [km-config](reference/km-config) - Keyman keyboards installation and information
+* [km-kvk2ldml](reference/km-kvk2ldml) - Convert a Keyman on-screen keyboard file to LDML
+* [km-package-get](reference/km-package-get) - Download a Keyman keyboard package
+* [km-package-install](reference/km-package-install) - Install a Keyman keyboard package
+* [km-package-list-installed](reference/km-package-list-installed) - List installed Keyman keyboard packages
+* [km-package-uninstall](reference/km-package-uninstall) - Uninstall a Keyman keyboard package
+
```
Comment (darcywong00) [inline linux/help/index.md:22]: Can remove l.30 now

Comment (mcdurdin) [reply linux/help/index.md:22]: I guess ... not important though? 😆

**Comment by ermshiperete** [comment]
> Query (@ermshiperete): are the docs accurate for 14.0 or have options changed?

`km-config` has changed slightly, so I regenerated the file for that.

**Comment by mcdurdin** [comment]
Is this a dup of #5762? Did I just miss that when I created this?

**Comment by darcywong00** [comment]
> Is this a dup of #5762? Did I just miss that when I created this?

Yeah, but since Eberhard regenerated the km-config reference on this PR, I think we close #5762

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.283](https://keyman.com/downloads/releases/stable/14.0.283)

**Pull request merged**